Description: Fix manpage.
 In particular, fix typos, escape hyphens, and change .SP to .Sp.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-14

--- a/wmrack.1
+++ b/wmrack.1
@@ -39,10 +39,10 @@
 .SH OPTIONS
 .TP
 \fB\-b\fP \fBCOLSPEC\fP or \fB\*=background\fP \fBCOLSPEC\fP
-Specifies the background color of the LED. It is avaiable through the symbolic 
+Specifies the background color of the LED. It is available through the symbolic
 XPM colorname
 .I led_color_black.
-.SP
+.Sp
 The other colors of the LED can be changed using the \fB-l\fP option.
 .Sp
 To find out what colors are available, try
@@ -59,7 +59,7 @@
 Specifies the color of the led. It is darkened two times (for a total of
 three colors) to get all needed colors. So better use a bright color.
 .Sp
-For the design of own XPM, the ledcolor is made avaiable through the use
+For the design of own XPM, the ledcolor is made available through the use
 of the symbolic xpm color names
 .I led_color_high,
 .I led_color_low,
@@ -116,9 +116,9 @@
 .Sp
         wmrack
         WMRack
-        \fIPOSITION\fP 2 -1 0
+        \fIPOSITION\fP 2 \-1 0
         wmrack
-        --withdrawn
+        \-\-withdrawn
 .Sp
 Another way to call \fBwmrack\fP is to swallow it into the \fBafterstep(1)\fP
 \fBWharf(1)\fP. This is best done placing the following line into your
@@ -130,7 +130,7 @@
 \fBFvwmButtons(1)\fP of the \fBfvwm(1)\fP window manager.
 .SH GENERAL USAGE
 The \fBwmrack\fP features a simple click interface. All times the buttons will
-show a standart sign which specifies the action it will do. But there are some 
+show a standard sign which specifies the action it will do. But there are some
 noteable extensions to that explained below.
 .Sp
 You can always change the controls shown by clicking the middle button (this
@@ -166,7 +166,7 @@
 The cd player also offers a play list functionality. By default it will map
 out all data tracks of the current cdrom, but you can also randomize the list
 or program it to the tracks you want to hear. These functions are only
-avaiable while a cdrom is in the drive and the drive is \fIstopped\fP.
+available while a cdrom is in the drive and the drive is \fIstopped\fP.
 .TP
 .B randomize
 Clicking Control-Button-3 to the track display causes the playlist to be
